README.md

```diff
@@ -0,0 +1,153 @@
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h3><b>Leaderboard</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 My capstone project <a name="about-project"></a>
+
+> This is a leaderboard website where user can input their name and score.
+
+
+**Leaderboard** is about implementing a new API
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Built with html, css, javascript, webpack.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.learn-html.org/">HTML</a></li>
+  </ul>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **adding menu bar**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link]()
```

Comment:
- [OPTIONAL] kindly consider removing this section if you decide to not deploy this app currently to avoid confusion to the people visiting your readme